### ISSUE TITLE
Keyboard navigation should be improved.

### ISSUE THREAD

**Issue by mikeamelin** [surveyjs/survey-library#6421]
![Image](https://user-images.githubusercontent.com/17025812/232199299-f9fcc608-3baa-4158-8443-c7875f5a7290.png)
